.github/workflows/maven-verify.yml

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Java CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest]
```

Comment:
I could see testing on Windows but is much gained by testing on MacOS? Should rarely differ from Linux behavior right?

Reply:
I tried but got this https://github.com/apache/maven-integration-testing/runs/6897982060?check_suite_focus=true 
and frankly as this project is build with maven core (e.g build with every branches/PRs) I don't want to waste too much time on a project already build a lot!

Reply:
windows added with Jenkinsfile